wordpress-mobile/WordPress-Android
# Feature/UI modernization posts and pages

**Placeholder PR** 

This PR is the feature branch of UI Modernization of Posts and Pages implementation. Read more about it [here](pcdRpT-3VB) 

## To Test
1. Login to the Jetpack app/Wordpress app
2. Go to Pages and Posts 
3. Verify the changes are as expected 
4. Verify the changes are as per the designs


## Regression Notes
1. Potential unintended areas of impact

5. What I did to test those areas of impact (or what existing automated tests I relied on)

6. What automated tests I added (or what prevented me from doing so)


PR submission checklist:

UI Changes testing checklist:
